WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyRing/PrivacyRingItemViewModel.cs

```diff
@@ -27,6 +27,21 @@ public PrivacyRingItemViewModel(PrivacyRingViewModel parent, SmartCoin coin, dou
 		AmountText = $"{Coin.Amount.ToFormattedString()} BTC";
 		Unconfirmed = !coin.Confirmed;
 		Confirmations = coin.GetConfirmations();
+
+		Reference =
+			this switch
+			{
+				{ IsPrivate: true } => "Private",
+				{ IsSemiPrivate: true } => "Semi private",
+				{ IsNonPrivate: true } => "Non private",
+				_ => ""
```

Comment:
I am not fully sure if this is better, at least shorter. Ignore if you want.

```suggestion
				{ IsPrivate: true } => "Private",
				{ IsSemiPrivate: true } => "Semi private",
				_ => "Non private"
```

Reply:
@soosr I decided to keep the current code and add `"Unknown"` for the `default` case. This way, we will quickly realize if there's ever a bug and none of the flags are setup (which shouldn't happen, and the user should never see `Unknown` in there.